public/index.html

```diff
@@ -4,7 +4,7 @@
     <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1,minimal-ui" />
     <link rel="shortcut icon" type="image/x-icon" href="https://decentraland.org/images/icons/favicon-32x32.png" />
-    <link rel="preload" href="%PUBLIC_URL%/images/background-v3@blur.jpg" as="image" />
+    <link rel="preload" href="/images/background-v3@blur.jpg" as="image" />
```

Comment:
When the site gets built, these images will be loaded from the CDN, right? If we use the relative path, it won't be able to find the image as the relative path from `decentraland.zone/play` will be different from the CDN site.